tests/integration/requirements.txt

```diff
@@ -1,2 +1,3 @@
 ansible==2.0.1
 dnspython==1.15.0
```

Comment:
This line can be removed here as well, since the ansible-load-balancer role was the only place where it was used.  Or you can leave it in, just in case we want to use it in future, up to you what you prefer.